cmd/validator/validator.go

```diff
@@ -20,7 +20,7 @@ optional flags:
   -output
      	Destination of a file to outputting results
   -reporter string
-    	Format of the printed report. Options are standard and json (default "standard")
+    	A comma separated list of format of the printed report. Options are standard, json and junit (default "standard")
```

Comment:
I would use an Oxford list

```suggestion
    	A comma separated list of format of the printed report. Options are standard, json, and junit (default "standard")
```